src/IdentityManager/WebApi/Extensions/ServiceConfigurationExtension.cs

```diff
@@ -0,0 +1,38 @@
+﻿using IdentityManager.WebApi.Filters;
+using IdentityManager.WebApi.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Travely.IdentityManager.API.Identity;
+using Travely.IdentityManager.Repository.Abstractions.Entities;
+
+namespace IdentityManager.WebApi.Extensions
+{
+    public static class ServiceConfigurationExtension
+    {
+        public static void ConfigureFilterServices(this IServiceCollection services)
+        {
+            services.AddControllers(config =>
+            {
+                config.Filters.Add(new ValidateModelStateActionFilter());
```

Comment:
I beleive ApiController attribute already do this for you

Reply:
I add this for swagger, to fix the error responseModel in UI